src/DynamoCoreWpf/Extensions/ViewLoadedParams.cs

```diff
@@ -69,14 +69,19 @@ public Window DynamoWindow
             dynamoMenu = dynamoView.titleBar.ChildOfType<Menu>();
             viewStartupParams = new ViewStartupParams(dynamoVM);
             DynamoSelection.Instance.Selection.CollectionChanged += OnSelectionCollectionChanged;
-
         }
 
         public void AddMenuItem(MenuBarType type, MenuItem menuItem, int index = -1)
         {
             AddItemToMenu(type, menuItem, index);
         }
 
+        // Adds the extension window to a new tab in the right side bar.
+        public void AddToExtensionsSideBar(Window w)
```

Comment:
I think the AC - propose to keep this internal for now - in addition this should not take a window but a UserControl I think, and definitely needs summary tags and param tags.

Reply:
Yes I changed it for some testing. Will make this internal and change the parameters.

Reply:
The API now takes in ContentControl object as the input parameter